components/my-account/User/UserForm/UserForm.tsx

```diff
@@ -0,0 +1,368 @@
+                />
+              )}
+            />
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            md={1.5}
+            sx={{ paddingTop: { xs: '10px !important', md: '64px' } }}
+            style={{ display: 'flex', alignItems: 'center' }}
+          >
+            <Controller
+              name="lastName"
+              control={control}
+              render={({ field }) => (
+                <KiboTextBox
+                  fullWidth
+                  error={!!errors?.lastName}
+                  value={field.value || ''}
+                  helperText={errors?.lastName?.message}
+                  label={t('last-name-or-sur-name')}
+                  onChange={(_name, value) => field.onChange(value)}
+                />
+              )}
+            />
+          </Grid>
+          <Grid item xs={12} md={2} sx={{ paddingTop: { xs: '10px !important', md: '64px' } }}>
+            <Controller
+              name="role"
+              control={control}
+              defaultValue={getValues('role')}
+              render={({ field }) => (
+                <KiboRadio
+                  {...field}
+                  align="center"
+                  onChange={(value) => field.onChange(value)}
+                  title={mdScreen ? '' : 'Role'}
+                  radioOptions={userFormRadioOptions}
+                  selected={getValues('role')}
+                />
+              )}
+            />
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            md={1}
+            sx={{ paddingTop: { sm: '0 !important', md: '50px !important' } }}
+          >
+            {isEditMode && (
+              <Controller
+                name="isActive"
+                control={control}
+                render={({ field }) => (
+                  <KiboSwitch
+                    checked={field.value}
+                    onLabel={t('active')}
+                    offLabel={t('in-active')}
+                    title={t('status')}
+                    onChange={(value) => field.onChange(value)}
+                  />
+                )}
+              />
+            )}
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            md={2.5}
+            sx={{ paddingTop: { xs: '10px !important', md: '64px' } }}
+            style={{ display: 'flex', alignItems: 'center', justifyContent: 'flex-end' }}
+          >
+            {isEditMode && mdScreen && (
+              <>
+                <Button
+                  variant="outlined"
+                  type="reset"
+                  onClick={cancelAction}
+                  style={{
+                    backgroundColor: theme.palette.grey[600],
+                    color: theme.palette.secondary.light,
+                    marginRight: '8px',
+                  }}
+                >
+                  <ClearIcon />
+                </Button>
+                <Button
```

Comment:
There should be a LoadingButton example somewhere. Use that

Reply:
No Loading button available